src/main/scala/edu/berkeley/ce/sparkrocks/SeedJointSelector.scala

```diff
@@ -160,12 +165,14 @@ object SeedJointSelector {
       } else {
         // Joint meets criteria
         val remainingVolume = jointOption.get
-        if (remainingVolume.volume <= volumePerPiece) {
+        val nonRedundantVolume = Block(remainingVolume.center, remainingVolume.nonRedundantFaces)
```

Comment:
Can you explain why you need to eliminate the non-redundant faces here? Does it really matter when we do this during the normal rock slicing process anyways?